core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -5203,7 +5203,7 @@ components:
         cloudPlatform:
           type: string
           description: The cloud platform of the workspace
-          enum: 
+          enum:
```

Comment:
I didn't intentionally change this, but I verified that Swagger still reads correctly with this change.